desktop/cmp/button/ZoneMapperButton.ts

```diff
@@ -69,11 +71,18 @@ export const [ZoneMapperButton, zoneMapperButton] = hoistCmp.withFactory<ZoneMap
                     zoneMapper({model: mapperModel})
                 ]
             }),
-            onInteraction: willOpen => {
-                if (willOpen) {
+            onInteraction: (willOpen, e?) => {
+                if (isOpen && !willOpen) {
+                    // Prevent clicks with Select controls from closing popover
+                    const selectPortal = document.getElementById(MENU_PORTAL_ID),
```

Comment:
yikes --  this could hit any application.  Is there a way we can change select to not leak these?

Reply:
Yeah, we have similar patterns in some client apps that would be great to unwind in favour of built-in protection. I'll file a ticket 👍